keps/sig-scheduling/5055-dra-device-taints-and-tolerations/README.md

```diff
@@ -227,14 +226,10 @@ the purpose of this KEP:
 - The ResourceClaim controller will remove such a completed pod from the claim's
   `ReservedFor` and deallocate the claim once it has no consumers.
 
-Taints are cumulative as long as the key and effect pairs are different:
-- Taints defined by an admin in a ResourceSlicePatch get added to the
+Taints are cumulative:
```

Comment:
The original KEP was following the node taints, which use key+effect as unique keys in the node spec slice, with some pretty confusing description.

But device taints don't need such a unique key (ResourceSlice spec has only a single writer, DeviceTaintRule only a single taint). Enforcing this uniqueness across slices and rules would have been more complicated than the "set of taints" approach, so the implementation did not follow the original KEP in this regard.